cumulus/parachains/runtimes/assets/asset-hub-westend/src/lib.rs

```diff
@@ -1180,6 +1181,9 @@ construct_runtime!(
 
 		StateTrieMigration: pallet_state_trie_migration = 70,
 
+		// Snowbridge
+		SnowbridgeSystemFrontend: snowbridge_pallet_system_frontend = 80,
```

Comment:
Maybe put this as 36 next to the other bridging stuff? Allocating a 10s chunk for one pallet is not so good.  
@claravanstaden can we please do this as follow-up? I am planning on using those ranges in another MR.

Reply:
@ggwpez added here: https://github.com/paritytech/polkadot-sdk/pull/8020/commits/08d64596ac04823d5b2ed69ff647bec8df24a297